code/datums/components/nanites.dm

```diff
@@ -24,7 +24,7 @@
 	if(isliving(parent))
 		host_mob = parent
 
-		if(!(MOB_ORGANIC in host_mob.mob_biotypes) && !(MOB_UNDEAD in host_mob.mob_biotypes) && !isipc(host_mob)) //Shouldn't happen, but this avoids HUD runtimes in case a silicon gets them somehow.
+		if(!(MOB_ORGANIC in host_mob.mob_biotypes) && !(MOB_UNDEAD in host_mob.mob_biotypes) && !((MOB_ROBOTIC in host_mob.mob_biotypes) && (MOB_HUMANOID in host_mob.mob_biotypes))) //Shouldn't happen, but this avoids HUD runtimes in case a silicon gets them somehow.
```

Comment:
```suggestion
		if(!(MOB_HUMANOID in host_mob.mob_biotypes)  || (MOB_INORGANIC in host_mob.mob_biotypes)) //Shouldn't happen, but this avoids HUD runtimes in case a silicon gets them somehow.
```

Reply:
honestly, might as well do this at this point

Reply:
Did you mean to use MOB_ORGANIC instead of MOB_INORGANIC? This takes away nanites from a bunch of simplemobs I think.

Reply:
ah, fair, did this while half asleep so I didn't quite consider simplemobs.
The idea seems to just be preventing silicons from getting nanites, so using `issilicon()` is probably the best course of action

Reply:
I used issilicon() for the nanite checks, is this resolved then?

Reply:
ah, yeah